submissions/YuraZagor/dom tsk/index.js

```diff
@@ -0,0 +1,57 @@
+import { data } from './data.js'  
+
+const body = document.querySelector('body')
+let text = data[0].content
+
+const div = document.createElement('div')
+div.classList.add('wrapper')
+
+body.appendChild(div)
+
+const main = document.createElement('main')
+main.classList.add('main')
+div.appendChild(main)	
+
+const header = document.createElement('header')
+header.classList.add('header')
+main.appendChild(header)
+
+const content = document.createElement('div')
+content.classList.add('content')
+main.appendChild(content)
+
+
+const footer = document.createElement('footer')
+footer.classList.add('footer')
+main.appendChild(footer)
+
+let li
+content.innerHTML = text
+
+const ul = document.createElement('ul')
+ul.classList.add('sidenav--ul')
+main.insertAdjacentElement('beforebegin', ul)
+
+data.forEach((datapiece)=> {
+	let sidenavLi = document.createElement('li')
+	sidenavLi.classList.add('sidenav--li')
+	
+	ul.appendChild(sidenavLi)
+	sidenavLi.setAttribute('id', datapiece.number-1)
```

Comment:
```suggestion
	sidenavLi.setAttribute('id', datapiece.id)
```

Reply:
Your logic is clear, but every time I try to implement it  I get "undefined" instead of the needed text and no errors in console.  will keep drilling it,   this writing is just to show I really apreciate your attention and I really work on it